parser.js/ParserJS.idr

```diff
@@ -11,18 +11,26 @@ import Backend.ReasonML
 
 generateCode : String -> (n ** TNamed n) -> String
 generateCode "haskell"  (n  **tn) = toString $ generateDefs Haskell tn
-generateCode "reasonml" (n  **tn) = toString $ generateDefs ReasonML tn
-generateCode "json"     (Z  **tn) = toString $ generate JSONDef tn
-generateCode "json"     (S _**tn) = "<error : can't generate JSON schema for open typedef>"
-generateCode _          _         = "<error : unknown backend>"
+generateCode _          _         = "<error : unsupported backend>"
 
 -- re-exports
 parseType : String -> Maybe (n : Nat ** TNamed n)
```

Comment:
We no longer export `parseType`, so this comment isn't accurate anymore, right?

Reply:
This is true. I'll remove it :)